src/app/core/data/request.models.ts

```diff
@@ -185,6 +186,12 @@ export class BrowseItemsRequest extends GetRequest {
   }
 }
 
+export class MappingCollectionsRequest extends GetRequest {
```

Comment:
Also needs Typedocs.  This should likely also be renamed "MappedCollectionsRequest" (the REST endpoint is now named `/mappedCollections`)